### PR TITLE
Feature/gsye 236

### PR DESCRIPTION
**Pull request by hannesdiedrich**
## Reason for the proposed changes

In order to better debug and follow the DOF algorithm ...

## Proposed changes

...  the matching requirements are added to the created Trade and TradeBidOfferInfo objects.

**INTEGRATION_TESTS_BRANCH**=master
**GSY_FRAMEWORK_BRANCH**=feature/GSYE-236

### CODE REVIEW

File: src/gsy_e/gsy_e_core/simulation.py [hunk 1]
```diff
@@ -588,7 +588,7 @@ def _execute_simulation(
 
                 # reset tick_resume after possible resume
                 tick_resume = 0
-                log.trace("Tick %s of %s in slot %s (%.1f%)", tick_no + 1,
+                log.trace("Tick %s of %s in slot %s (%.1f)", tick_no + 1,
```
Comment (spyrostz) [inline src/gsy_e/gsy_e_core/simulation.py:591]: How come ? I think this log really intends to have a percentage value. Might it be that there was an exception raised and we had not seen it due to the trace level ?

File: src/gsy_e/models/market/two_sided.py [hunk 2]
```diff
@@ -230,7 +236,7 @@ def accept_bid(self, bid: Bid,
         if energy > market_bid.energy:
             raise InvalidTrade(f"Traded energy ({energy}) cannot be more than the "
                                f"bid energy ({market_bid.energy}).")
-        elif energy < market_bid.energy:
+        if energy < market_bid.energy:
```
Comment (spyrostz) [inline src/gsy_e/models/market/two_sided.py:239]: I guess pylint right ? Otherwise the statement looks correct to me as it was.

**Review by spyrostz — APPROVED** [review]
Some clarification comments, LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Review by Hassan754 — APPROVED** [review]

**Pull request merged**